sources/packages/backend/libs/integrations/src/institution-integration/ece-integration/ece-response.processing.service.ts

```diff
@@ -390,6 +417,86 @@ export class ECEResponseProcessingService {
     }
   }
 
+  /**
+   * Create ECE Response file processing notifications.
+   * @param institutionCode institution of the processing file.
+   * @param disbursementProcessingDetails disbursement processing count.
+   * @param processSummaryResult process summary details.
+   */
+  private async createECEResponseProcessingNotification(
+    institutionCode: string,
+    disbursementProcessingDetails: DisbursementProcessingDetails,
+    processSummaryResult: ProcessSummaryResult,
+  ): Promise<void> {
+    try {
+      const integrationContacts =
+        await this.institutionLocationService.getIntegrationContactsByInstitutionCode(
+          institutionCode,
+        );
+
+      // Create email notifications only if integration contacts are available.
+      if (integrationContacts.length) {
+        const notification: ECEResponseFileProcessingNotification = {
+          institutionCode,
+          integrationContacts,
+          fileParsingErrors: disbursementProcessingDetails.fileParsingErrors,
+          totalRecords: disbursementProcessingDetails.totalRecords,
+          totalDisbursements: disbursementProcessingDetails.totalDisbursements,
+          disbursementsSuccessfullyProcessed:
+            disbursementProcessingDetails.disbursementsSuccessfullyProcessed,
+          disbursementsSkipped:
+            disbursementProcessingDetails.disbursementsSkipped,
+          duplicateDisbursements:
+            disbursementProcessingDetails.duplicateDisbursements,
+          disbursementsFailedToProcess:
+            disbursementProcessingDetails.disbursementsFailedToProcess,
+          attachmentFileContent:
+            this.buildEmailAttachmentBody(processSummaryResult),
+        };
+
+        await this.notificationActionsService.saveECEResponseFileProcessingNotification(
+          notification,
+        );
+        processSummaryResult.summary.push(
+          "Notification has been created to send email to integration contacts.",
+        );
+      } else {
+        processSummaryResult.summary.push(
+          "Notification cannot be created as no integration contacts found for the institution.",
+        );
+      }
+    } catch (error: unknown) {
+      this.logger.error(error);
+      processSummaryResult.errors.push(
+        "Unexpected error while creating notifications.",
+      );
+    }
+  }
+
+  /**
+   * Build the email attachment content from process summary.
+   * @param processSummaryResult process summary.
+   * @returns email attachment body.
+   */
+  private buildEmailAttachmentBody(
+    processSummaryResult: ProcessSummaryResult,
+  ): string {
+    const attachmentBody = new StringBuilder();
+    attachmentBody.appendLine("Summary:");
+    attachmentBody.appendLine(processSummaryResult.summary.join(END_OF_LINE));
+    const warnings = processSummaryResult.warnings.length
+      ? processSummaryResult.warnings.join(END_OF_LINE)
+      : "NONE";
```

Comment:
@dheepak-aot sorry for asking this now, can u pls share the email attachment screenshot, I did;t able to find it in the PR description

Reply:
No problem. I am going to update PR description.

Reply:
I updated the email attachment. Please have a look.